src/signals/incident/containers/IncidentContainer/reducer.js

```diff
@@ -13,34 +13,37 @@ import {
   CREATE_INCIDENT_ERROR,
 
   GET_CLASSIFICATION_SUCCESS,
-  GET_CLASSIFICATION_ERROR
+  GET_CLASSIFICATION_ERROR,
+
+  SET_PRIORITY,
+  SET_PRIORITY_SUCCESS,
+  SET_PRIORITY_ERROR
 } from './constants';
 
 export const initialState = fromJS({
   incident: {
     // location: {
```

Comment:
Can we move this test state to a separate file?

Reply:
gedaan